HPXMLtoOpenStudio/resources/hvac_sizing.rb

```diff
@@ -1566,6 +1583,242 @@ def self.process_equipment_adjustments(hvac_final_values, weather, hvac)
     end
   end
 
+  def self.process_sizing_for_installation_quality(hvac_final_values, weather, hvac, model)
+    # Increases the autosized heating/cooling capacities to account for any reduction
+    # in capacity due to HVAC installation quality. This is done to prevent causing
+    # unmet loads.
+
+    return unless hvac.has_type(Constants.ObjectNameCentralAirConditioner) || hvac.has_type(Constants.ObjectNameAirSourceHeatPump) || hvac.has_type(Constants.ObjectNameMiniSplitHeatPump) || hvac.has_type(Constants.ObjectNameGroundSourceHeatPump)
```

Comment:
@yzhou601 Should, e.g., a standalone furnace with an airflow defect be included in here? Check the base-hvac-install-qual-all-furnace-gas-only.xml test file.

Reply:
I guess my question also applies to the heating side of an AC+furnace system.

Reply:
The coil capacity is not degraded for furnace, should we adjust furnace sizing for airflow defect? @jonwinkler

Reply:
Jon: "no, the furnace modeling (capacity or efficiency) should not be affected by the airflow defect. the heating capacity is fixed, and when the airflow is reduced the supply air temperature increases."

So we are good here.